WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeButtonPrefsFragment.java

```diff
@@ -0,0 +1,347 @@
+package org.wordpress.android.ui.publicize;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.android.volley.VolleyError;
+import com.wordpress.rest.RestRequest;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wordpress.android.R;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.fluxc.Dispatcher;
+import org.wordpress.android.fluxc.model.SiteModel;
+import org.wordpress.android.models.PublicizeButton;
+import org.wordpress.android.ui.prefs.SiteSettingsInterface;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.NetworkUtils;
+import org.wordpress.android.util.StringUtils;
+import org.wordpress.android.util.ToastUtils;
+import org.wordpress.android.widgets.WPPrefView;
+import org.wordpress.android.widgets.WPPrefView.PrefListItem;
+import org.wordpress.android.widgets.WPPrefView.PrefListItems;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+import javax.inject.Inject;
+
+import static android.view.View.GONE;
```

Comment:
Bit of a nitpick, but since these constants are only used once and we already import `View` I think we should access them via `View.GONE/View.VISIBLE`.